coast/NEMO.py

```diff
@@ -615,6 +615,26 @@ def differentiate(self, in_varstr, dim='z_dim', out_varstr=None, out_obj=None):
             warn(f"{in_varstr} does not exist in {get_slug(self)} dataset")
             return None
         
+    def apply_doodson_xo_filter(self, var_str):
+        ''' Applies Doodson XO filter to a variable. 
+    
+        Input variable is expected to be hourly.
+        Output is saved back to original dataset as {var_str}_dxo
+        
+        !!WARNING: Will load in entire variable to memory. If dataset large,
+        then subset before using this method or ensure you have enough free 
+        RAM to hold the variable (twice). 
+        
+        DB:: Currently not tested in unit_test.py'''
```

Comment:
I think it is now, right?

Reply:
No, NEMO.apply_dooson_xo_filter isn't tested, only TIDEGAUGE.apply_doodson_xo_filter. There isn't enough time steps in the example NEMO data to test it at the moment. I've added this as an issue though.